Categories/Category/Product.agda

```diff
@@ -135,9 +135,10 @@ _※ⁿⁱ_ : ∀ {D₁ : Category o ℓ e} {D₂ : Category o′ ℓ′ e′}
 
 module _ (C₁ : Category o ℓ e) (C₂ : Category o′ ℓ′ e′) (C₃ : Category o″ ℓ″ e″) where
 
-  module C₁ = Category C₁
-  module C₂ = Category C₂
-  module C₃ = Category C₃
+  private
```

Comment:
This is a nice clean up, but I prefer if orthogonal clean-ups are in their own PRs. Those will get merged in way faster.

Reply:
Done: now in #71.